test/mocks.ts

```diff
@@ -14,19 +14,9 @@ export const coreMock = {
 
 export function mockInputs(options: Options) {
   coreMock.getInput.mockImplementation(name => {
-    const option = options[name as keyof Options];
-    if (option === undefined) {
-      throw new Error(`Unexpected input: ${name}`);
```

Comment:
This is a more accurate representation of the real behavior: https://github.com/actions/toolkit/blob/415c42d27ca2a24f3801dd9406344aaea00b7866/packages/core/src/core.ts#L120